src/anaconda_ai.pth

```diff
@@ -0,0 +1 @@
+import os, sys;exec('try:\n    import litellm\n    from anaconda_ai.integrations.litellm import AnacondaLLM\n    anaconda_llm = AnacondaLLM()\n    litellm.custom_provider_map.append(\n        {"provider": "anaconda", "custom_handler": anaconda_llm}\n    )\nexcept ImportError:\n    pass\n')
```

Comment:
Why do we need this as a .pth hook? Might be worth some followup

Reply:
This .pth is need until litellm merges my PR (or their own implementation) for entry-point support

https://github.com/BerriAI/litellm/pull/15881